modules/ROOT/pages/functions/temporal/duration.adoc

```diff
@@ -33,6 +34,7 @@ See also xref::values-and-types/temporal.adoc[Temporal values] and xref::express
 | The values of the parameters may be arbitrarily large.
 | The values of the parameters may be negative.
 | The xref:values-and-types/temporal.adoc#cypher-temporal-accessing-components-durations[components of `DURATION` objects] are individually accessible.
+| The `pattern` parameter is constructed from xref:functions/temporal/format.adoc#query-functions-temporal-format-duration-types-characters[Allowed characters for duration types].
```

Comment:
I wonder if we want to call it 'pattern argument' rather than 'pattern parameter', as in the table above it is referred to as an argument? If you decide to change this, it also applies to other parts of the PR.

A few of the bullet points above mentions parameter, but they seem to be about the values of the map/the numeric parts of the string in the input argument. It is not really part of this PR but I also feel like that use of the world parameter is also a bit confusing for a user, maybe 'component' would be better (that is what we call it in Example 1)?